testing/citest/tests/google_front50_test.py

```diff
@@ -63,8 +63,8 @@ def _do_init_bindings(self):
     """Hook to initialize custom test bindings so journaling is scoped."""
     context = ExecutionContext()
     config = self.agent.deployed_config
-    enabled = config.get('spinnaker.gcs.enabled', False)
-    if not enabled:
+    enabled = config.get('spinnaker.gcs.enabled', None)
+    if enabled == False:
       raise ValueError('spinnaker.gcs.enabled is not True')
 
     self.BUCKET = self.bindings['GCS_BUCKET'] or config['spinnaker.gcs.bucket']
```

Comment:
Since you arent guaranteeing config, you should check for BUCKET to fail fast. If there is no BUCKET, I'd treat the whole test as misconfigured similar to GCS not being enabled.